### DISCUSSION TITLE
How to connect your 1 Vhost with youre ques

### DISCUSSION THREAD

**Discussion by cagi89** [rabbitmq/rabbitmq-server#10851] [Questions] [answered]
Hi guys

i have a question i hope you can help me. I connect my Rabbitmq with my Actice Directory and everything works now but not correct. 

My Problem is i need ONE  VHOST and i have 5 Teams. All of them should produce ques and exchanges but they don´t have to see each others ques. 

I found this page but it doesnt work for me 
https://github.com/MarcialRosales/rabbitmq-ldap-integration/blob/master/auth-and-authz/README.md

**Comment by michaelklishin** [comment, marked as answer]
Virtual hosts are not "connected" to queues, they contain queues, and act as a [level of logical separation](https://www.rabbitmq.com/docs/vhosts).

When [LDAP](https://www.rabbitmq.com/docs/ldap) is used, a dedicated query will be issued by the plugin with the name of the virtual host. Then it's up to your LDAP tree and query to only allow access to the given virtual host to a suitable/authorized group of users.
